tasks.py

```diff
@@ -411,6 +411,67 @@ def cleanup(c):
         OUTDIR.rmdir()
         message("Ok", level=Level.SUCCESS)
 
+@task
+def start_flower(c, node=None, all_=False, flower=True):
+    """ Remove existing flower container, start monitoring tools """
+
+    kill_flower(c)
+    message("Flower has withered away", Level.HEADER)
+
+    node_ids = []
+    if all_:
+        for node_config_file in listdir(NODES_CONFIG_DIR):
+            filename = Path(node_config_file).stem
+            node_ids.append(filename)
+    elif node:
+        node_config_file = NODES_CONFIG_DIR / f"{node}.toml"
+        if not Path(node_config_file).is_file():
+            message(
+                f"The configuration file for node '{node}', does not exist in directory '{NODES_CONFIG_DIR}'",
+                Level.ERROR,
+            )
+            sys.exit(1)
+        filename = Path(node_config_file).stem
+        node_ids.append(filename)
+    else:
+        message("Please specify a node using --node <node> or use --all", Level.WARNING)
+        sys.exit(1)
+
+    for node_id in node_ids:
+        node_config_file = NODES_CONFIG_DIR / f"{node_id}.toml"
+        with open(node_config_file) as fp:
+            node_config = toml.load(fp)
+
+        ip = node_config['rabbitmq']['ip']
+        port = node_config['rabbitmq']['port']
+        user_and_password = node_config['rabbitmq']['user'] + ":" + node_config['rabbitmq']['password']
+        vhost = node_config['rabbitmq']['vhost']
+        flower_url = ip + ":" + str(port)
+        broker_api = f"amqp://{user_and_password}@{flower_url}/{vhost}"
+
+        flower_index = node_ids.index(node_id)
+        flower_port = 5555 + flower_index
+
+        message(f"Starting flower container for node {node_id}...", Level.HEADER)
+        command = f"docker run --name flower{flower_index} -d -p {flower_port}:5555 mher/flower:0.9.5 flower --broker={broker_api} "
+        run(c, command)
+        cmd = "docker ps | grep '[f]lower'"
+        run(c, cmd, warn=True)
```

Comment:
To put my OCD at rest, I just wanted to mention you could use `run(c,cmd,show_ok=False)` when a message has not been printed before to avoid:
![image](https://user-images.githubusercontent.com/15667989/118971851-901d8580-b978-11eb-8c91-e002b2a79343.png)

The `run` command will always print `OK` unless told otherwise.